### PR TITLE
Add December 2019 slide deck

### PR DESCRIPTION
**Pull request by WyriHaximus**
Todo:
- [x] Speaker info
- [x] Hoster info
- [x] Next hoster info
- [x] Conferences and events

### CODE REVIEW

File: 2019-December.html [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+        <!--[if lt IE 9]>
+        <script src="lib/js/html5shiv.js"></script>
+        <![endif]-->
+
+        <script>
+            slideData = {
+                "text": {
+                    "date": "December 2019",
+                    "speaker-name": "Gabriel Caruso",
```
Comment (lcobucci) [inline 2019-December.html]:
```suggestion
                    "speaker-name": "Stacy Cashmore",
```

File: 2019-December.html [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+                    <h4>May 21th-23th</h4>
+                    <h4>Belgrade, Serbia</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">PHP Day</h1>
+                    <img width="600" src="img/conference-logos/phpday-2017.svg">
+                    <h4>May 14th-15th</h4>
+                    <h4>Verona, Italy</h4>
+                </section>
```
Comment (lcobucci) [inline 2019-December.html]: Please swap these two to keep things in chronological order :grimacing:

File: img/conference-logos/php-wales-2020.svg [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (lcobucci) [inline img/conference-logos/php-wales-2020.svg]: You added the logo but I don't see the conference on the list

Comment (WyriHaximus) [reply img/conference-logos/php-wales-2020.svg]: Gabby told me to pull it because the list go to long :zipper_mouth_face:

Comment (lcobucci) [reply img/conference-logos/php-wales-2020.svg]: It's alright :+1:

File: 2019-December.html (hunk 2 above)
Comment (WyriHaximus) [reply 2019-December.html]: Done

**Comment by WyriHaximus** [comment]
:tada:

**Pull request merged**